lib/utils/properties-panel.js

```diff
@@ -384,7 +384,11 @@ export function getErrorMessage(id, report) {
     return 'Type must be defined.';
   }
 
-  if (id === 'messageSubscriptionCorrelationKey') {
+  if (id === 'messageSubscriptionCorrelationKey'
+    && [
+      ERROR_TYPES.EXTENSION_ELEMENT_REQUIRED,
```

Comment:
How is this related to the PR description?

Reply:
The check against the ID of the properties panel entry is now too generic since there is more than one error type for the _Subscription correlation key_ entry. We have to do the same for [other entries](https://github.com/camunda/linting/blob/main/lib/utils/properties-panel.js#L447) as well.